### DISCUSSION TITLE
filedragdrop, how do I influence my UI with dragn drop?

### DISCUSSION THREAD

**Discussion by WyvernIXTL** [DioxusLabs/dioxus#1893] [Q&A] [answered]
I don't understand the example `filedragdrop.rs` in examples.

```rust
use dioxus::prelude::*;
use dioxus_desktop::Config;
use dioxus_desktop::wry::webview::FileDropEvent;

fn main() {
    let cfg = Config::new().with_file_drop_handler(|_w, e| {
        match e {
                FileDropEvent::Dropped { paths, .. } => { // here I get path when user drops file },
                _ => {}
        };
        true
    });

    dioxus_desktop::launch_with_props(app, (), cfg);
}

fn app(cx: Scope) -> Element {
    render! {
        // my ui
        // how do I influence it with the file path?
    }
}
```

The code in #95 does not work for me.

**Comment by ealmloff** [comment, marked as answer]
If you would like to accept a file on a specific element, a file input might be better. See this example: https://github.com/DioxusLabs/dioxus/blob/417cea75697dcf010c8d0e73454caee28a190272/examples/file_upload.rs#L34

You can use a hover selector in css to change the appearance of the element while you hover over it

**Reply by WyvernIXTL** [reply]
I see. Thanks.
